### DISCUSSION TITLE
missing updates in character LCD w. snapclient

### DISCUSSION THREAD

**Discussion by xmabri** [rern/rAudio#772] [Fixed Issues] [answered]
rAudio-Client (20220117) running as snapcast client with lcd character display didn't update if song changes on server. I have to stop streaming and restart snapclient, than time and trackname will be updated on lcd.
if I run /srv/http/bash/status-push.sh on console, than display will be updated too.
It seems that status-push-sh will only be called once at snapclient start and  so lcd shows wrong info after end of first track streamed.
Of Cause, I'm far away from understanding the whole code ;-)
Can you pse have a look at this bug - I'm think that's no feature ;-)

with regards 
Martin

**Comment by rern** [comment, marked as answer]
Fixed.

**Comment by xmabri** [comment]
Hi ren,
just make a short test - good job! Display get updates as snapcast client streaming from NAS and from webradio (running on server) too.
Mark Problem as solved!

regards
Martin
